flowmachine/tests/test_model.py

```diff
@@ -0,0 +1,208 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+Tests for models.
+"""
+from typing import List
+
+import pandas as pd
+import pytest
+
+from flowmachine.core import Query
+from flowmachine.core.model import Model, model_result
+from flowmachine.core.model_result import ModelResult
+
+
+class DummyQuery(Query):
```

Comment:
Not a serious issue, but is it worth renaming this to avoid confusion with `flowmachine.core.dummy_query.DummyQuery`?

Reply:
We have purpose specific `DummyQuery` classes in a few places, so I'm not super worried about that.